modules/common/data/vehicle_param.pb.txt

```diff
@@ -1,48 +1,21 @@
 vehicle_param {
-  brand: LINCOLN_MKZ
```

Comment:
Modifications to this file do not have to be submitted

Reply:
noted, thanks.